src/ui/components/results/paginationcomponent.js

```diff
@@ -91,7 +92,12 @@ export default class PaginationComponent extends Component {
      * @type {string}
      * @private
      */
-    this._pageLabel = config.pageLabel === undefined ? 'Page' : config.pageLabel;
+    this._pageLabel = config.pageLabel !== undefined
```

Comment:
I think we may need to make an additional item for internationalizing pagination. Right now, the component creates text of the form `${_pageLabel}: ${pageNum}`. This assumes that all languages follow the `label: num` syntax and I'm not sure that's true. I think we may need to do some interpolation here.

Reply:
Could you clarify slightly more what you're seeing? 

In the component's .hbs file, it has the form `[pageLabel] [pageNum]`, for example `Page 1` (no colon, code here: https://github.com/yext/answers/blob/feature/localized-assets/src/ui/templates/results/pagination.hbs#L60). On some non-english search engines, I've seen the label "Page" omitted entirely and this JS currently supports that use-case. Is the concern that some languages may use a different convention entirely (e.g. `1 Page`, or `Page #1`)?

Reply:
If it's a convention thing, I can totally add it to an interpolation helper - like "[pageLabel] [pageNum]" - in this PR https://github.com/yext/answers/pull/1057